### ISSUE TITLE
Indicate that the configure process needs to be executed with stack-build

### ISSUE THREAD

**Issue by msoos** [xiw/stack#2]
In the README it's not indicated that the configuration process should be done with stack-build. For cmake, simply issuing:

``` bash
$ ~/my-path-to-stack-build/stack-build make
```

will simply recompile with already-selected compiler. I believe this is the same for configure-based systems, though I could be mistaken. For cmake builds, the following is needed:

``` bash
$ ~/my-path-to-stack-build/stack-build cmake
$ make
```

Notice that with cmake, I don't need to use stack-build when issuing make.

**Comment by zeldovich** [comment]
Thanks, will fix.

**Comment by xiw** [comment]
fixed in f1f8e067, thanks.
